### PR TITLE
Added useful links

### PR DESCRIPTION
**Pull request by ashwini0529**

### CODE REVIEW

File: Python/Useful_links.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Name | Links 
+------------ | ------------- 
+Python Audio Video Material | https://www.python.org/doc/av/
+-Networking in Pytho | http://www.tutorialspoint.com/python/python_networking.htm
```
Comment (PareshMayani) [inline Python/Useful_links.md]: - is there and typo in python spelling

**Comment by PareshMayani** [comment]
And please follow [this format](https://github.com/GDGAhmedabad/Learning-Resources/blob/master/android/Useful_links.md) for sharing resources/links/materials, sorry for creating confusion!

**Comment by ashwini0529** [comment]
Okay. Sorry, I didn't notice.

**Comment by PareshMayani** [comment]
Also please put some details/brief about links, it would be helpful when I am going through the list of links!

**Comment by ashwini0529** [comment]
Okay. :+1:

**Pull request closed without merging**